examples/quickstarts/helidon-quickstart-se/src/main/java/io/helidon/examples/quickstart/se/GreetService.java

```diff
@@ -122,7 +122,14 @@ private void updateGreetingFromJson(JsonObject jo, ServerResponse response) {
      */
     private void updateGreetingHandler(ServerRequest request,
                                        ServerResponse response) {
-        request.content().as(JsonObject.class).thenAccept(jo -> updateGreetingFromJson(jo, response));
+        request.content()
+               .as(JsonObject.class)
+               .thenAccept(jo -> updateGreetingFromJson(jo, response))
+               .exceptionally(t -> {
+                   JsonObject jsonErrorObject = JSON.createObjectBuilder().add("error", "No body provided").build();
```

Comment:
The `exceptionally` branch is indeed needed, though it should not disregard the `throwable` (or it should have a more general content).

It just prints "No body provided` and `BAD_REQUEST` status - yet the throwable may be a `NullPointerException`, `CancelledException` or basically any other uncaught exception thrown while processing the future, or an explicitly set throwable when completing the future exceptionally. 


Options:

1. Send "Internal server error" (500) and error with text "request failed" or similar and log the message
2. Check exception type (you would need to actually check the cause, as CompletionStage exceptions are wrapped) and create a customized message and status code depending on the failure; and log the message
3. Use the exception provided (`t`) and write its stacktrace to the response - this is only for debugging/development purposes (and should be documented so)

Reply:
Hi, you have marked this as resolved, though the code is not changed. I think both @romain-grecourt and me requested a very similar change.

Reply:
Hi @tomas-langer, thanks for mentioning this. I accidentally resolved your request and will make the changes as soon as possible. Thanks for being patient with my first PR.